core/chain_util.go

```diff
@@ -42,6 +42,9 @@ var (
 
 	ExpDiffPeriod = big.NewInt(100000)
 	blockHashPre  = []byte("block-hash-") // [deprecated by eth/63]
+
+	mipmapPre    = []byte("mipmap-bloom-")
```

Comment:
Just a thought, but maybe I'd name the prefix "receipt-bloom" or something similar, otherwise if we decide to add another set of bloom filters, we'll have to rename this and do a db upgrade.